### DISCUSSION TITLE
en/latest/walk-through/dag/

### DISCUSSION THREAD

**Discussion by giscus** [argoproj-labs/hera#823] [Website Discussions]
# en/latest/walk-through/dag/

https://hera-workflows.readthedocs.io/en/latest/walk-through/dag/

**Comment by nambrosini17** [comment]
Thanks for the page, really useful. Just once piece of constructive feedback. Make more emphasis on the fact that "Dependencies can be described over multiple statements" starts a new example independent of the previous one. I spent some time thinking it was wrong until i figure out it was just another example of dependency between tasks

**Reply by flaviuvadan** [reply]
Thank you for the feedback @nambrosini17! Concretely, you're suggesting that we can do something like 
```python
a = foo(...)
b = bar(...)
a >> b

c = bee(...)
d = gee(...)
b >> c
b >> d
```

That's how I understood your suggestion of `Dependencies can be described over multiple statements `

**Reply by nambrosini17** [reply]
That's not exactly what i meant. Let me share a screenshot of what i mean. 
![image](https://github.com/argoproj-labs/hera/assets/146840853/fb24a179-06f2-4755-bb89-64ffb0457e57)

Due to screen size (and perhaps a lack of emphasis) it isn't obvious that after the diagram **we're presenting a new example** when listing the bullets, and not referring to the diamond diagram above. 

If you scroll down the new diagram is visible so my suggestion would be to rephrase

> Dependencies can be described over multiple statements 

To something like

> To give another example, dependencies can be described over multiple statements

Or something like that. English is not my native language so perhaps you can rephrase but i believe some formatting would help understand it's a new example, not the same one as before

**Reply by flaviuvadan** [reply]
Got it, see https://github.com/argoproj-labs/hera/pull/824

**Reply by elliotgunton** [reply]
Rereading the feedback, we missed in #824 the idea that the next example isn't introduced as a different dag. Will make a one liner to the doc.

**Comment by flaviuvadan** [comment]
Closing as https://github.com/argoproj-labs/hera/pull/824 addresses this. Please feel free to reopen if needed!
